core/api/src/main/java/org/trellisldp/api/Metadata.java

```diff
@@ -45,11 +46,12 @@
      * @param memberRelation an LDP hasMemberRelation predicate, may be {@code null}
      * @param memberOfRelation an LDP isMemberOfRelation predicate, may be {@code null}
      * @param insertedContentRelation an LDP insertedContentRelation, may be {@code null}
+     * @param revision a revision value, may be {@code null}
```

Comment:
How about a Javadoc link to the doc for `Resource::getRevision`? I think that would help people reading the code.

Reply:
Good idea.